anta/cli/get/commands.py

```diff
@@ -106,6 +106,24 @@ def from_ansible(ctx: click.Context, output: Path, ansible_group: str, ansible_i
         ctx.exit(ExitCode.USAGE_ERROR)
 
 
+@click.command
+@click.pass_context
+@inventory_output_options
+@click.option("--nb-instance", "-nbi", help="Name of the NetBox instance", type=str, required=True)
+@click.option("--nb-token", "-nbt", help="NetBox token", type=str, required=True)
+@click.option("--nb-platform", "-nbp", help="NetBox device platform", type=str, default="Arista EOS", required=False)
+@click.option("--nb-site", "-nbs", help="NetBox site (case sensitive)", type=str, default=None, required=False)
+@click.option("--nb-verify", "-nbf", help="NetBox verify SSL", type=bool, default=False, required=False)
```

Comment:
1/ Given that you are in the from-netbox command context I dont think you need to add `nb` prefix to the options (except if it makes a lot of sense)

2/ the default for security should always be `True` ;)

3/ I think (and we may not be doing this everywhere) that you don't need `required=False` as it is the default in click. Same for `default=None`